config/locales/en.yml

```diff
@@ -1,5 +1,6 @@
 en:
   errors:
+    incorrect_password: is incorrect
```

Comment:
The error message in the logs, that is. On the web page, the error message appears below the field that it is applied to, and just says "is incorrect". This is the convention in Simple Form.

See this screenshot:

<img width="1280" alt="screen shot 2016-09-02 at 3 23 08 pm" src="https://cloud.githubusercontent.com/assets/811150/18215816/2b0b933e-7121-11e6-9a1f-636de2b64dee.png">